src/app/main/pages/budgets/budgets/budgets-pending-send/dialogs/pending-send-timeline-dialog/pending-send-timeline-dialog.component.html

```diff
@@ -0,0 +1,17 @@
+<div class="container">
+  <h2 mat-dialog-title class="card">
+    <div class="card--title" mat-dialog-close>
+      <mat-icon mat-list-icon>timeline</mat-icon>
+    </div>
+    Historial
+    <span class="ms-fill"></span>
+    <div mat-dialog-close class="card--title__close--button">
+      <mat-icon>clear</mat-icon>
+    </div>
+  </h2>
```

Comment:
@carlos-dubon-ms 
Ya tenemos clases creadas para las cabeceras de los diálogos.
```suggestion
  <div class="ms-dialog-header">
        <mat-icon class="ms-dialog-header__icon">timeline</mat-icon>
        <h3 class="ms-dialog-header__title">Historial</h3>
        <span class="ms-fill"></span>
        <button mat-icon-button mat-dialog-close>
            <mat-icon>cancel</mat-icon>
        </button>
  </div>
```